pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -177,16 +177,14 @@ Configuration parse(List<String> args) => _Parser(args).parse();
 PathConfiguration _parsePathConfiguration(String option) {
   final uri = Uri.parse(option);
 
-  final name = uri.queryParameters['name'];
+  final names = uri.queryParametersAll['name'];
   final fullName = uri.queryParameters['full-name'];
```

Comment:
Since we won't combine these (which I think is reasonable) we should probably throw if both are passed.